airqo-platform/AirQo-api
# [mobile-app] send welcome messages

**_WHAT DOES THIS PR DO?_**

- [x] update the content  of the welcome messages
- [x] Add Kafka to auth service
- [x] Modify the auth service to listen to the Kafka topic (`new-mobile-app-user-topic`) and extract the new user details.
- [x] Send a welcome message to the new users.


**_WHAT ISSUES ARE RELATED TO THIS PR?_**

- Jira cards
    - [MOB-78]

**_HOW DO I TEST OUT THIS PR?_**
Ensure that you have all the necessary ENVs with their corresponding values, some of which are listed below:

> - UNIQUE_CONSUMER_GROUP=
> - UNIQUE_PRODUCER_GROUP=
> - NEW_MOBILE_APP_USER_TOPIC=new-mobile-app-user-topic
> - KAFKA_BOOTSTRAP_SERVERS_DEV=localhost:9092
> - KAFKA_CLIENT_ID_DEV=
> - KAFKA_CLIENT_GROUP_DEV=
> - KAFKA_BOOTSTRAP_SERVERS_STAGE=
> - KAFKA_CLIENT_ID_STAGE=
> - KAFKA_CLIENT_GROUP_STAGE=
> - KAFKA_BOOTSTRAP_SERVERS_PROD=
> - KAFKA_CLIENT_ID_PROD=
> - KAFKA_CLIENT_GROUP_PROD=
> - KAFKA_BOOTSTRAP_SERVERS_PROD=
> - KAFKA_CLIENT_ID_PROD=
> - KAFKA_CLIENT_GROUP_PROD=

```
cd src/auth-service
npm install
npm run stage-mac
```

**_ANY IMAGE DESCRIPTION OF PR?_**
![Screenshot 2023-03-14 at 20 19 52](https://user-images.githubusercontent.com/1590213/225086793-1cdd40b6-11ac-4108-9a3d-a2e6871cecee.png)





[MOB-78]: https://airqoteam.atlassian.net/browse/MOB-78?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
